plasmapy/formulary/collisions.py

```diff
@@ -2033,3 +2038,178 @@ def coupling_parameter(
+
+        Examples
+        --------
+        >>> import astropy.units as u
+        >>> T_a = 1 * u.eV
+        >>> n_b = 1e20 * u.cm**-3
+        >>> T_b = 1e3 * u.eV
+        >>> Coulomb_log = 10 * u.dimensionless_unscaled
+        >>> frequencies = CollisionFrequencies(
+        ...     "e-", "e-", T_a=T_a, n_b=n_b, T_b=T_b, Coulomb_log=Coulomb_log
+        ... )
+        >>> frequencies.slowing_down
+        <Quantity 1.83701432e+11 Hz>
+
+        See Also
+        --------
+        ~plasmapy.formulary.collisions.Coulomb_logarithm : Evaluates the Coulomb
+            logarithm for two interacting electron species.
+
+        ~plasmapy.formulary.collisions.collision_frequency : The Lorentz collision frequency.
+        """
+
+        if v_a is None:
+            if T_a is not None:
+                v_a = thermal_speed(T_a, test_particle).to(u.cm / u.s)
+            else:
+                raise ValueError("Please specify either v_a or T_a.")
+        elif T_a is not None:
+            raise ValueError("Please specify either v_a or T_a, not both.")
+
+        self.test_particle = test_particle
+        self.field_particle = field_particle
+        self.v_a = v_a
+        self.T_a = (
+            T_a
+            if T_a is not None
+            else (0.5 * test_particle.mass.to(u.g) * v_a**2 / k_B).to(u.K)
+        )
+        self.n_b = n_b
+        self.T_b = T_b
+        self.Coulomb_log = Coulomb_log
+
+        x = self._x()
+        phi = self._phi(x)
+        phi_prime = self._phi_prime(x)
+
+        mass_ratio = test_particle.mass / field_particle.mass
+
+        self.v_0 = self.Lorentz_collision_frequency()
+        self.slowing_down = (1 + mass_ratio) * phi * self.v_0
+        self.transverse_diffusion = 2 * ((1 - 1 / (2 * x)) * phi + phi_prime) * self.v_0
+        self.parallel_diffusion = (phi / x) * self.v_0
+        self.energy_loss = 2 * (mass_ratio * phi - phi_prime) * self.v_0
+
+    def Lorentz_collision_frequency(self):
+        r"""
+        The Lorentz collision frequency (see Ch. 5 of :cite:t:`chen:2016`) is given
+        by
+
+        .. math::
+
+            ν = n σ v \ln{Λ}
+
+        where :math:`n` is the particle density, :math:`σ` is the
+        collisional cross-section, :math:`v` is the inter-particle velocity
+        (typically taken as the thermal velocity), and :math:`\ln{Λ}` is the
+        Coulomb logarithm accounting for small angle collisions.
+
+        See Equation (2.14) in :cite:t:`callen:unpublished`.
+        """
+
+        return (
+            4
+            * np.pi
+            * (self.test_particle.charge_number * e.esu) ** 2
+            * (self.field_particle.charge_number * e.esu) ** 2
+            * self.Coulomb_log
+            * self.n_b
+            / (self.test_particle.mass.to(u.g) ** 2 * self.v_a**3)
+        ).to(u.Hz)
+
+    def _x(self) -> u.dimensionless_unscaled:
```

Comment:
Include a brief 1-sentence docstring on these private methods, just for future coders reference. Also, x could be a property and might actually be good to expose to the users and document fully: that way they can see which regime the are in.